app/controllers/api/v1/subnets_controller.rb

```diff
@@ -25,6 +25,7 @@ def show
       api :POST, '/subnets', 'Create a subnet'
       param :subnet, Hash, :required => true do
         param :name, String, :desc => 'Subnet name', :required => true
+        param :network_type, String, :desc => 'Type or protocol, IPv4 or IPv6, defaults to IPv4'
```

Comment:
Instead of String, use `Subnet::SUBNET_TYPES.values` - if you give an array then it acts like an enumeration, suggesting the user can only pass the listed values.